recipes/glew/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "glew" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "cc88d2f70a185185eb050eb13789f498425fcd58230bfd65da496b8008f046f2" %}
+{% set dev_url = "https://github.com/nigels-com/glew" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: {{ dev_url }}/releases/download/{{ name }}-{{ version }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+  patches:
+    - CMakeLists.patch  # [win and (py27 or py34)]
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - toolchain
+    - cmake
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+
+# Tests require visual context not present on CIs
+test:
+  requires:
+    - cmake
+    - toolchain
+  commands:
+    - test -f $PREFIX/include/GL/glew.h  # [unix]
+    - if not exist %LIBRARY_INC%\GL\glew.h exit 1  # [win]
+
+about:
+  home: http://glew.sourceforge.net/
+  license: BSD
```

Comment:
It at least needs `BSD 3-Clause`. I don't know enough about the other license or this library to advise you on whether those other licenses need to be included or not. The link for the GPL part appears to point to the GPL2 license.